01_Data/src/layers/sequelize/repository/Authorization.ts

```diff
@@ -131,46 +131,9 @@ export class SequelizeAuthorizationRepository extends SequelizeRepository<Author
     return await super.deleteAllByQuery(this._constructQuery(query), Authorization.MODEL_NAME);
   }
 
-  /**
-   * Private Methods
-   */
 
-  private _constructQuery(queryParams: AuthorizationQuerystring): object {
-    return {
-      where: {},
-      include: [
-        {
-          model: IdToken,
-          where: { idToken: queryParams.idToken, type: queryParams.type },
-          required: true, // This ensures the inner join, so only Authorizations with the matching IdToken are returned
-        },
-        { model: IdTokenInfo, include: [{ model: IdToken, include: [AdditionalInfo] }] },
-      ],
-    };
-  }
-
-  private _createInclude(value: AuthorizationData): BuildOptions {
-    const include: Includeable[] = [];
-    if (value.idTokenInfo) {
-      const idTokenInfoInclude: Includeable[] = [];
-      if (value.idTokenInfo.groupIdToken) {
-        const idTokenInfoGroupIdTokenInclude: Includeable[] = [];
-        if (value.idTokenInfo?.groupIdToken.additionalInfo) {
-          idTokenInfoGroupIdTokenInclude.push(AdditionalInfo);
-        }
-        idTokenInfoInclude.push({ model: IdToken, include: idTokenInfoGroupIdTokenInclude });
-      }
-      include.push({ model: IdTokenInfo, include: idTokenInfoInclude });
-    }
-    const idTokenInclude: Includeable[] = [];
-    if (value.idToken.additionalInfo) {
-      idTokenInclude.push(AdditionalInfo);
-    }
-    include.push({ model: IdToken, include: idTokenInclude });
-    return { include };
-  }
 
-  private async _updateIdToken(value: IdTokenType, transaction?: Transaction): Promise<IdToken> {
+  async updateIdToken(value: IdTokenType, transaction?: Transaction): Promise<IdToken> {
```

Comment:
Why make it public instead of private? Still used only inside the class

Reply:
Oops, didn't mean to keep that change. Fixed!